pkg/workceptor/kubernetes.go

```diff
@@ -127,8 +128,16 @@ func (kw *kubeUnit) createPod(env map[string]string) error {
 			return fmt.Errorf("At least one container must be named worker")
 		}
 		spec.RestartPolicy = corev1.RestartPolicyNever
+		userNamespace := pod.ObjectMeta.Namespace
+		if userNamespace != "" {
```

Comment:
This allows for omitting the `namespace` attribute in the receptor config. It will also override any value present in the config.